pinodo/coffee-mathcer
# feat: animate sunny background

### Summary
Added a sunny background animation

### Steps to test  
In `App.jsx`, set weather to `Weathers.SUNNY` and make sure you see the rotating sun animation
